portability/toku_pthread.cc

```diff
@@ -67,7 +67,9 @@ int toku_pthread_yield(void) {
 #elif defined(HAVE_PTHREAD_YIELD_NP)
     pthread_yield_np();
     return 0;
+#elif defined(HAVE_SCHED_YIELD)
+    return sched_yield();
 #else
-# error "cannot find pthread_yield or pthread_yield_np"
+# error "cannot find pthread_yield or pthread_yield_np or sched_yield"
```

Comment:
Please, maybe we should consider `sched_yield` as the first option as the others are deprecated?

Reply:
Well, it doesn't really matter as this storage is going to be discontinued

Reply:
@ldonoso I just wanted to be less intrusive, so that we won't change anything for existing platforms where `pthread_yield()` / `pthread_yield_np()` are not considered deprecated.